test/Chainweb/Test/Utils.hs

```diff
@@ -544,19 +548,26 @@ withChainwebTestServer tls v appIO envIO test = withResource start stop $ \x ->
         close sock
 
 clientEnvWithChainwebTestServer
-    :: Show t
+    :: forall t cas
+    .  Show t
     => ToJSON t
     => FromJSON t
     => PayloadCas cas
     => Bool
     -> ChainwebVersion
-    -> IO (ChainwebServerDbs t logger cas)
+    -> IO (ChainwebServerDbs t GenericLogger cas)
```

Comment:
Why is this change needed?

Reply:
To massage the types in the right way. I remember it getting very mad at me that it couldn't resolve a `logger` instance. Let me double-check.

Reply:
Yes, it was to get things compiling. I believe that in the old case there was no `Logger =>` constraint on that `logger` type variable, but now there is. Some other test utility function was forcing this to be `()`, which clearly doesn't have the instance. So I hardcode it to `GenericLogger` here, since that's what in scope.